README.md

```diff
@@ -100,6 +105,7 @@ const config = {
 const conn = connect(config)
 const results = await conn.execute('select 1 from dual')
 console.log(results)
+await disconnectAll()
```

Comment:
I think we should leave the undici default recommendation in place and add this example in a section below. Then clients have an easy default recommendation to use and are also aware of the `fetch-h2` option with the extra `disconnectAll` requirement.

We've used undici in production for years and have good experience with it. I haven't used fetch-h2, but I was happy to learn about it here.

Reply:
I was shocked to learn that unidici doesn't do h2. So what do you recommend instead? http/1.1 for this case is pretty not ideal, we shouldn't be using that at all.

Reply:
There's also this old open issue: https://github.com/nodejs/undici/issues/399

node core is also... very lacking and slowly working towards getting support for h3 in stdlib, which would need similar plumbing to support, since it seems while stdlib has support for h2, they don't use it for their `fetch()`, and I'm not sure why, which is a bit frustrating and kinda defeats the purpose. But I guess this is how node ecosystem works.

I'd strongly prefer we push boundaries here if we can to yield better performance for a database driver.